### PR TITLE
Added documentation about BaseX user management / permissions

### PR DESCRIPTION
**Pull request by Flurb**

### CODE REVIEW

**Review by Sander3003 — APPROVED** [review]
Looks good.

**Review by Sander3003 — DISMISSED** [review]
Looks good to me!

File: DATABASE_MANAGEMENT.md [hunk 1]
```diff
@@ -82,6 +82,43 @@ A node in this case is an XML node like an element, attribute, text, etc.
...
+[Example with xQuery](https://code-repo.d4science.org/gCubeSystem/d4science-keycloak-themes/src/branch/master/src/utils/xquery)
+Example with RESTXQ is not available yet. The RESTXQ module, in combination with the basex:perm annotation, protects the access to a GUI implementing the OAuth2 Code-grant flow. This is definitely something we can use.
+
+### Is direct database access allowed within the microservices architecture?
+For maintenance for example, it's of course allowed to have direct database access. There is no best practice available for this. For some things, you just need direct database access.
+
+If other microservices need access to the data of an other microservice, the only way (best practice) to do this is by API calls.
+
+Source:
+https://microservices.io/patterns/data/database-per-service.html
```
Comment (Sander3003) [inline DATABASE_MANAGEMENT.md:120]: "Each service has database credentials that only grant it access its own (logical) database on a shared MySQL server."

Is this the setup we want to do here with BaseX?

**Review by Sander3003 — APPROVED** [review]
Looks good

File: DATABASE_MANAGEMENT.md (hunk 1 above)
Comment (Flurb) [reply DATABASE_MANAGEMENT.md:128]: No, each service has it's own database.

But parts of the database can be enclosed for certain API calls, which can be done by credentials / certain users.

**Review by Sander3003 — DISMISSED** [review]

**Review by Sander3003 — APPROVED** [review]
Looks good!

**Pull request merged**